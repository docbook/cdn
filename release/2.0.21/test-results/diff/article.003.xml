<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: article.003.xml</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="article" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: article.003.xml</h2>
            <div class="authorgroup">
               <div class="author">
                  <h3>
                     <span class="personname">
                        <span class="firstname">Norman</span>
                        <span class="surname">Walsh</span>
                     </span>
                  </h3>
               </div>
               <div class="author">
                  <h3>
                     <span class="personname">
                        <span class="firstname">Jane</span>
                        <span class="surname">Doe</span>
                     </span>
                  </h3>
               </div>
            </div>
            <div class="abstract">
               <p>This is the abstract.</p>
               <p>It has several paras.</p>
               <p>It has several paras.</p>
            </div>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#article.3">1. Section One</a>
                     </span>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#article.3.3">1.1. Section One.Two</a>
                           </span>
                        </li>
                        <li>
                           <span>
                              <a href="#article.3.4">1.2. Section One.Three</a>
                           </span>
                        </li>
                     </ul>
                  </li>
                  <li>
                     <span>
                        <a href="#article.4">2. Section Two</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#article.5">A. Appendix One</a>
                     </span>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#article.5.3">1. Section One</a>
                           </span>
                           <ul class="toc">
                              <li>
                                 <span>
                                    <a href="#article.5.3.3">1.1. Section One.Two</a>
                                 </span>
                              </li>
                              <li>
                                 <span>
                                    <a href="#article.5.3.4">1.2. Section One.Three</a>
                                 </span>
                              </li>
                           </ul>
                        </li>
                     </ul>
                  </li>
               </ul>
            </div>
         </div>
         <p deltaxml:deltaV2="A">Some preambulatory paragraph.</p><div deltaxml:deltaV2="B" class="content">
            <p>Some preambulatory paragraph.</p>
            <section id="article.3" class="section">
               <div class="section-titlepage">
                  <h2>1. Section One</h2>
               </div>
               <div class="content">
                  <p>Some other paragraph.</p>
                  <section id="article.3.3" class="section">
                     <div class="section-titlepage">
                        <h3>1.1. Section One.Two</h3>
                     </div>
                     <div class="content">
                        <p>Some other paragraph.</p>
                     </div>
                  </section>
                  <section id="article.3.4" class="section">
                     <div class="section-titlepage">
                        <h3>1.2. Section One.Three</h3>
                     </div>
                     <div class="content">
                        <p>Some other paragraph.</p>
                     </div>
                  </section>
               </div>
            </section>
            <section id="article.4" class="section">
               <div class="section-titlepage">
                  <h2>2. Section Two</h2>
               </div>
               <div class="content">
                  <p>Some other paragraph.</p>
               </div>
            </section>
            <article id="article.5" class="appendix">
               <header class="appendix-titlepage">
                  <h2>A. Appendix One</h2>
               </header>
               <div class="lists-of-titles">
                  <div class="toc">
                     <p>
                        <b>Table of Contents</b>
                     </p>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#article.5.3">1. Section One</a>
                           </span>
                           <ul class="toc">
                              <li>
                                 <span>
                                    <a href="#article.5.3.3">1.1. Section One.Two</a>
                                 </span>
                              </li>
                              <li>
                                 <span>
                                    <a href="#article.5.3.4">1.2. Section One.Three</a>
                                 </span>
                              </li>
                           </ul>
                        </li>
                     </ul>
                  </div>
               </div>
               <div class="content">
                  <p>Some other paragraph.</p>
                  <section id="article.5.3" class="section">
                     <div class="section-titlepage">
                        <h2>1. Section One</h2>
                     </div>
                     <div class="content">
                        <p>Some other paragraph.</p>
                        <section id="article.5.3.3" class="section">
                           <div class="section-titlepage">
                              <h3>1.1. Section One.Two</h3>
                           </div>
                           <div class="content">
                              <p>Some other paragraph.</p>
                           </div>
                        </section>
                        <section id="article.5.3.4" class="section">
                           <div class="section-titlepage">
                              <h3>1.2. Section One.Three</h3>
                           </div>
                           <div class="content">
                              <p>Some other paragraph.</p>
                           </div>
                        </section>
                     </div>
                  </section>
               </div>
            </article>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="article.3" class="section">
            <div class="section-titlepage">
               <h2>1. Section One</h2>
            </div>
            <p>Some other paragraph.</p>
            <section id="article.3.3" class="section">
               <div class="section-titlepage">
                  <h3>1.1. Section One.Two</h3>
               </div>
               <p>Some other paragraph.</p>
            </section>
            <section id="article.3.4" class="section">
               <div class="section-titlepage">
                  <h3>1.2. Section One.Three</h3>
               </div>
               <p>Some other paragraph.</p>
            </section>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="article.4" class="section">
            <div class="section-titlepage">
               <h2>2. Section Two</h2>
            </div>
            <p>Some other paragraph.</p>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><article deltaxml:deltaV2="A" id="article.5" class="appendix">
            <header class="appendix-titlepage">
               <h2>A. Appendix One</h2>
            </header>
            <div class="lists-of-titles">
               <div class="toc">
                  <p>
                     <b>Table of Contents</b>
                  </p>
                  <ul class="toc">
                     <li>
                        <span>
                           <a href="#article.5.3">1. Section One</a>
                        </span>
                        <ul class="toc">
                           <li>
                              <span>
                                 <a href="#article.5.3.3">1.1. Section One.Two</a>
                              </span>
                           </li>
                           <li>
                              <span>
                                 <a href="#article.5.3.4">1.2. Section One.Three</a>
                              </span>
                           </li>
                        </ul>
                     </li>
                  </ul>
               </div>
            </div>
            <p>Some other paragraph.</p>
            <section id="article.5.3" class="section">
               <div class="section-titlepage">
                  <h2>1. Section One</h2>
               </div>
               <p>Some other paragraph.</p>
               <section id="article.5.3.3" class="section">
                  <div class="section-titlepage">
                     <h3>1.1. Section One.Two</h3>
                  </div>
                  <p>Some other paragraph.</p>
               </section>
               <section id="article.5.3.4" class="section">
                  <div class="section-titlepage">
                     <h3>1.2. Section One.Three</h3>
                  </div>
                  <p>Some other paragraph.</p>
               </section>
            </section>
         </article>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>