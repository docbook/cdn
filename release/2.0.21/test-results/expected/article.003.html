<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: article.003.xml</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="article" class="article"><header class="article-titlepage"><h2>Unit Test: article.003.xml</h2><div class="authorgroup"><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><div class="author"><h3><span class="personname"><span class="firstname">Jane</span> <span class="surname">Doe</span></span></h3></div></div><div class="abstract"><p>This is the abstract.</p><p>It has several paras.</p><p>It has several paras.</p></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#article.3">1. Section One</a></span><ul class="toc"><li><span><a href="#article.3.3">1.1. Section One.Two</a></span></li><li><span><a href="#article.3.4">1.2. Section One.Three</a></span></li></ul></li><li><span><a href="#article.4">2. Section Two</a></span></li><li><span><a href="#article.5">A. Appendix One</a></span><ul class="toc"><li><span><a href="#article.5.3">1. Section One</a></span><ul class="toc"><li><span><a href="#article.5.3.3">1.1. Section One.Two</a></span></li><li><span><a href="#article.5.3.4">1.2. Section One.Three</a></span></li></ul></li></ul></li></ul></div></div><p>Some preambulatory paragraph.</p><section id="article.3" class="section"><div class="section-titlepage"><h2>1. Section One</h2></div><p>Some other paragraph.</p><section id="article.3.3" class="section"><div class="section-titlepage"><h3>1.1. Section One.Two</h3></div><p>Some other paragraph.</p></section><section id="article.3.4" class="section"><div class="section-titlepage"><h3>1.2. Section One.Three</h3></div><p>Some other paragraph.</p></section></section><section id="article.4" class="section"><div class="section-titlepage"><h2>2. Section Two</h2></div><p>Some other paragraph.</p></section><article id="article.5" class="appendix"><header class="appendix-titlepage"><h2>A. Appendix One</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#article.5.3">1. Section One</a></span><ul class="toc"><li><span><a href="#article.5.3.3">1.1. Section One.Two</a></span></li><li><span><a href="#article.5.3.4">1.2. Section One.Three</a></span></li></ul></li></ul></div></div><p>Some other paragraph.</p><section id="article.5.3" class="section"><div class="section-titlepage"><h2>1. Section One</h2></div><p>Some other paragraph.</p><section id="article.5.3.3" class="section"><div class="section-titlepage"><h3>1.1. Section One.Two</h3></div><p>Some other paragraph.</p></section><section id="article.5.3.4" class="section"><div class="section-titlepage"><h3>1.2. Section One.Three</h3></div><p>Some other paragraph.</p></section></section></article></article><script src="../../resources/js/prism.js"></script></body></html>